src/test/data/caseChallengedAccessLogs.json

```diff
@@ -1,29 +1,25 @@
 {
   "accessLog": [
-    {      
-        "caseId": 123,
+    {
         "caseRef": "123",
         "userId": "456",
         "requestType": "challenged",
-        "timestamp": "2020-07-20T15:00:00",
+        "timestamp": "2020-07-20 15:00:00",
```

Comment:
I believe, returned results from backend do have `T` separating date and time. The same for all other timestamps

Reply:
Addressed, the issue was caused by not updating Logs.csvData()